pkg/driver/docker/docker.go

```diff
@@ -199,6 +199,11 @@ func (d *dockerDriver) RunDockerDevContainer(
 		"--sig-proxy=false",
 	}
 
+	// add environment variables that signals that we are in a remote container
+	// (vscode compatibility) and specifically that we are using devpod.
+	args = append(args, "--env", "DEVPOD=true")
+	args = append(args, "--env", "REMOTE_CONTAINERS=true")
```

Comment:
If you set this in `driver.RunOptions.Env` it'll also work for custom drivers

Reply:
Done :+1: